src/test/java/com/wikia/webdriver/elements/mercury/components/Sidebar.java

```diff
@@ -0,0 +1,147 @@
+package com.wikia.webdriver.elements.mercury.components;
```

Comment:
Can me move this three components for infobox builder to `infoboxbuilder` package?